operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -167,6 +175,21 @@ local utils = import 'mixin-utils/utils.libsonnet';
   recordingRulePrefix(selectors)::
     std.join('_', [matcher.label for matcher in selectors]),
 
+  // Overrite upstream's "stack" field to make it compatible with timeseriesPanel.
```

Comment:
Can you expand this comment a bit. It took me a file to figure out why this was required since we're using `$.timeseriesPanel()` in a few places below. I.e. explain that we're using some utility methods for panels from `grafana-builder/grafana.libsonnet` that can't be converted without coordinating updates to a bunch of different mixins in `jsonnet-libs`.

Reply:
🆙
I've rephrased the comment. Could you have another look, if it's less obscure now

Reply:
Looks good. In the future, please don't rebase / force-push after review. It makes it harder to figure out which changes are new since last time.